hokusai-app-updater/update_apps.sh

```diff
@@ -23,8 +23,6 @@ function prep() {
 
   echo "### stash ###"
   git stash
-  echo "### git pull ###"
-  git pull
```

Comment:
If my other working directories had a non-main branch checked out (or maybe if they weren't tracking an upstream branch), this step would error and interrupt the script. Since there's already a `stash` above and a `pull` below after checking out `main`, I think it can be skipped?

Reply:
oh. i [added](https://github.com/artsy/opstools/pull/4/files#diff-49a9fb6c263e406068d29bd02f78629d9f8413837f9c5cc675600c8c37cb96c9R26) this `git pull` b/c my local repo didn't have `main` branch (pre `master` -> `main` era). so this pull is to fetch `main` branch. maybe safe to omit now? or change to `get fetch`?

Reply:
If you don't have a local `main` branch, is `git fetch` actually enough to then call `git checkout main`?

Reply:
It appears so.
```
➜  acceleration git:(test) git branch
* test
➜  acceleration git:(test) git fetch
...
   d22ee0c..7a192d7  main                        -> origin/main
...
➜  acceleration git:(test) git checkout main
Branch 'main' set up to track remote branch 'main' from 'origin'.
Switched to a new branch 'main'
➜  acceleration git:(main) git log
commit 7a192d7ea3eff27317d737dd318bd2e277d5191d (HEAD -> main, origin/main, origin/HEAD)
...
```

Reply:
Works for me. Updated!

Reply:
thanks y'all !